frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/virtual-hardware-tab/virtual-hardware-tab.tsx

```diff
@@ -86,7 +94,7 @@ const VirtualHardwareTabFirehose: React.FC<VirtualHardwareTabFirehoseProps> = ({
   const diskWrapper = DiskWrapper.initializeFromSimpleData({
     name: availableCDName,
     type: DiskType.CDROM,
-    bus: templateValidations.getDefaultBus(),
+    bus: os?.startsWith('win') ? DiskBus.SATA : templateValidations.getDefaultBus(),
```

Comment:
so this will be later deprecated in favor of common templates right? Can you please write a TODO comment somewhere with the PR link: https://github.com/kubevirt/common-templates/pull/148. Together with bugzilla bug to track this.

Reply:
can we do toLowerCase and includes instead?